include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -728,15 +740,17 @@ __parallel_radix_sort(_ExecutionPolicy&& __exec, _Range&& __in_rng, _Proj __proj
     else if (__n <= 16384 && __wg_size * 8 <= __max_wg_size)
         __event = __subgroup_radix_sort<_RadixSortKernel, __wg_size * 8, 32, __radix_bits, __is_ascending>{}(
             __exec.queue(), ::std::forward<_Range>(__in_rng), __proj);
-    else if (__n <= 32768 && __wg_size * 16 <= __max_wg_size)
-        __event = __subgroup_radix_sort<_RadixSortKernel, __wg_size * 16, 32, __radix_bits, __is_ascending>{}(
-            __exec.queue(), ::std::forward<_Range>(__in_rng), __proj);
     else
     {
         constexpr ::std::uint32_t __radix_iters = __get_buckets_in_type<_KeyT>(__radix_bits);
         const ::std::uint32_t __radix_states = 1 << __radix_bits;
 
-        const ::std::size_t __wg_size = __max_wg_size;
+#if _ONEDPL_RADIX_WORKLOAD_TUNING
+        const auto __wg_sz_k = __n >= (1<<15)/*32K*/ && __n <  (1<<19)/*512K*/ ? 8 : __n <= (1<<21)/*2M*/ ? 4 : 1;
```

Comment:
Here is a suggestion according to clang-format:
```suggestion
        const auto __wg_sz_k =
            __n >= (1 << 15) /*32K*/ && __n < (1 << 19) /*512K*/ ? 8 : __n <= (1 << 21) /*2M*/ ? 4 : 1;
```